apps/docs/app/(home)/MyRuntimeProvider.tsx

```
@@ -5,18 +5,49 @@
+    yield token + " ";
+  }
+}
+
+const LOREM_IPSUM = `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.`;
+
+class DummyChatAdapter implements ChatModelAdapter {
+  async *run({ messages, abortSignal }: ChatModelRunOptions) {
+    const text =
+      messages.length === 1
+        ? "This is a mocked chat endpoint for testing purposes. \n\n" +
+          LOREM_IPSUM
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)